src/system/RoutesModule/Form/Handler/Route/EditHandler.php

```diff
@@ -12,76 +12,65 @@
 
 namespace Zikula\RoutesModule\Form\Handler\Route;
 
-use ModUtil;
 use Symfony\Component\Routing\RouteCollection;
 use Zikula\RoutesModule\Entity\RouteEntity;
 use Zikula\RoutesModule\Form\Handler\Route\Base\EditHandler as BaseEditHandler;
-use Zikula\RoutesModule\Routing\Util as RoutingUtil;
-use Zikula_Form_View;
 
 /**
- * This handler class handles the page events of the Form called by the zikulaRoutesModule_admin_edit() function.
+ * This handler class handles the page events of the Form called by the zikulaRoutesModule_route_edit() function.
  * It aims on the route object type.
  */
 class EditHandler extends BaseEditHandler
 {
-    public function initialize(Zikula_Form_View $view)
+    /**
+     * {@inheritdoc}
+     */
+    public function applyAction(array $args = [])
     {
-        $items = [];
-        $urlNames = [];
-        $modules = ModUtil::getModulesByState(3, 'displayname');
-        foreach ($modules as $module) {
-            $items[] = [
-                'text' => $module['displayname'],
-                'value' => $module['name']
-            ];
-            $urlNames[$module['name']] = $module['url'];
+        $this->sanitizeInput();
+        if ($this->hasConflicts()) {
+            return false;
         }
-        $view->assign('modules', $items);
-        $view->assign('moduleUrlNames', $urlNames);
```

Comment:
Moved to https://github.com/zikula/core/blob/1.4/src/system/RoutesModule/Helper/ViewHelper.php#L44 and https://github.com/zikula/core/blob/1.4/src/system/RoutesModule/Form/Type/RouteType.php#L34